opm/material/components/Brine.hpp

```diff
@@ -149,22 +149,22 @@ class Brine : public Component<Scalar, Brine<Scalar, H2O> >
                                      const Evaluation& pressure)
     {
         // Numerical coefficents from Palliser and McKibbin
-        static const Scalar f[] = {
+        static constexpr Scalar f[] = {
             2.63500e-1, 7.48368e-6, 1.44611e-6, -3.80860e-10
         };
 
         // Numerical coefficents from Michaelides for the enthalpy of brine
-        static const Scalar a[4][3] = {
+        static constexpr Scalar a[4][3] = {
             { -9633.6, -4080.0, +286.49 },
             { +166.58, +68.577, -4.6856 },
             { -0.90963, -0.36524, +0.249667e-1 },
             { +0.17965e-2, +0.71924e-3, -0.4900e-4 }
         };
 
-        const Evaluation& theta = temperature - 273.15;
+        const Evaluation theta = temperature - 273.15;
```

Comment:
Now I'm curious, why  is this a dangling reference. Shouldn't the lifetime be extended until theta goes out of scope?
That being  said, I don't oppose the change as the performance impacted is expected to be minimal.

Reply:
uhm, you are correct. those commits are just noise, not sure where my head was at. i can remove them.

Reply:
i am however quite certain they do no harm, as surely these will be moved.